### DISCUSSION TITLE
`React Server Components` category in `Discussions` is deleted

### DISCUSSION THREAD

**Discussion by bayraak** [vercel/next.js#48732] [Help]
Just a few days ago `React Server Components` category in Discussions was showing up fine. But currently, I am not able to find it.

The link to it was: https://github.com/vercel/next.js/discussions/categories/react-server-components

1)  Is there any specific reason why it is deleted?

There were lots of discussions in that category but everything disappeared.

2) How is Vercel planning to keep the community transparent while doing such actions?

3) Can we say that; Vercel doesn't like complainings about React Server Components and their failure to have it with `Next.JS 13`?

**Comment by icyJoseph** [comment]
I think it is now called, https://github.com/vercel/next.js/discussions/categories/app-router App Router, not sure why they did that, but I'd guess that tag is more of a React thing I'd say, in Next.js we use App Router, which implements React Server Components.

**Reply by bayraak** [reply]
It was totally a business decision of Vercel.

All issues, complainings were in that category. Including mine one.

Now, everything is deleted.

Hiding of the problems will not make things better. Next.js 13 is a mess and failure.

**Reply by icyJoseph** [reply]
How do you know all of this? Did they also delete your discussions? I searched for discussions opened by you, and found only two. 

I can see that discussions nominally opened under React Server Components still exist under `App Router`

I don't think the label change would imply deletion of comments. Have you got links to those? Is there any in particular you are looking for? According to the github API you have only two comments ever made here.

**Reply by bayraak** [reply]
Yes, my discussion is deleted in that category. After 2-3 days of opening a topic about how big a disaster is Next.js 13 and the way of using the React Server Components inside it, they deleted the whole category.

Probably they moved those topics that don't expose the mistakes they made.

The framework that was capable of everything, now become a big mess that doesn't work with anything together. The new version should fit with the majority of tools that are used widely. But for everything you have to do explicit configuration, workaround or you just deal with problems.

Apple changed the whole industry by changing the Processor architecture, it was a smooth change. 

In Next.js 13:

- CSS in JS don't work
- The themes and design system don't work. Migration is hell, some of them (MIU) use @emotion/styled. It is not possible to migrate because @emotion/styled doesn't work.
- TypeScript problems
- Hydration problems
- Even some Browser Plugins can show errors in your app
- File structuring is a mess; loading.tsx, 404.tsx, etc
- Socket.io doesn't work
- `node:crypto` in middleware doesn't work
- `use-client` `use-whatever`. Sounds like old ASPX pages. `runat="server"` `runat"client"`
- Learning curve increased 10x 
- Big projects are looking like a mess. You don't know what happens where. You have to be very careful.
- Lot's of other problems


They never tested it as it should. They rushed because "business" wants to deliver. They messed up everything.

Next.JS 13 is not production ready. They named it and tagged it with the major version name. Now let's watch disasters.
